src/eko/matching_conditions/as2.py

```diff
@@ -458,11 +450,11 @@ def A_singlet(n, sx, L, is_msbar=False):
         A_gq : :math:`A_{gq, H}^{S,(2)}`
         A_gg : :math:`A_{gg, H}^{S,(2)}`
     """
-    A_hq_2 = A_hq_ps(n, sx, L)
-    A_qq_2 = A_qq_ns(n, sx, L)
-    A_hg_2 = A_hg(n, sx, L)
-    A_gq_2 = A_gq(n, sx, L)
-    A_gg_2 = A_gg(n, sx, L)
+    A_hq_2 = A_hq_ps(n, sx[1, 0], L)
```

Comment:
Re "unique signature": I think I prefer the old code - in this case I would sacrifice the aim of "only take those arguments you truly need" over "unique signature"; here you already did compute all elements in `sx` so the expensive cost has already been paid ... what do you think?

Reply:
Okay, we can leave it as it was before...